hack/lib/const.sh

```diff
@@ -46,3 +46,5 @@ readonly CAPACT_USE_TEST_SETUP="false"
 #
 
 readonly CAPACT_INCREASE_RESOURCE_LIMITS="true"
+readonly CAPACT_HUB_MANIFESTS_SOURCE_REPO_URL="github.com/capactio/hub-manifests"
+readonly CAPACT_HUB_MANIFESTS_SOURCE_REPO_REF="5464edbf0b3f5ef2d1a02c6266dfc50fb5ac0a4a"
```

Comment:
Let's add some comment here about how to get this value

Reply:
TBH I don't understand what kind of comment I can put here. It's mapped to those values:
https://github.com/capactio/capact/blob/78dc5c67e81ac6eba32e07a41a8c35d4239ffe81/deploy/kubernetes/charts/capact/charts/hub-public/values.yaml#L68-L71

I added description of a new flag in related PR: https://github.com/capactio/website/pull/46/files#diff-30642f79ee617e8a1dc888506e5cb08d4173a93d28f08006e1ffb5929cee644a

could you suggest a comment that I can put here? Thanks in advance.

Reply:
I just want to add a comment how to get the value `5464edbf0b3f5ef2d1a02c6266dfc50fb5ac0a4a`  and when it needs to be updated.